vcx/wrappers/node/test/suite2/ffi.test.ts

```diff
@@ -20,7 +20,7 @@ describe('vcxInit', () => {
 // these tests were created to only test that the ffi could be called with each function
 
 describe('Using the vcx ffi directly', () => {
-  const path = '/usr/lib/libvcx.so'
+  const path = '/usr/local/lib/libvcx.so'
   const run = new VCXRuntime({ basepath: path })
```

Comment:
our CI runs vcx python tests on Ubuntu only. I believe we have to leave `/usr/lib` or to build it properly base on os.

Reply:
reverted